process/track/blockProcessor.go

```diff
@@ -311,14 +311,19 @@ func (bp *blockProcessor) getHighestRoundInReceivedHeaders(
 }
 
 func (bp *blockProcessor) requestHeadersIfNothingNewIsReceived(
+	lastNotarizedHeader data.HeaderHandler,
```

Comment:
lastNotarized header is not needed here, just the lastNotarizedHeader.GetNonce(), just the nonce could be passed as param.

Reply:
done

Reply:
ok